wynnset/tapestry-wp
# 533 - If user is able to add/edit, they should automatically be able to view that node too

Closes #533

**Testing**

* Uncheck the read permission if it is currently selected
* Check the add or edit permission
    * The read permission should be checked and the checkbox disabled
* Uncheck the add or edit permission
    * The read permission should be unchecked
* Check the read permission
* Check then uncheck the add or edit permission
    * The read permission should be unaffected
